commit 8882d4f278f00f73e898542740fbba6614ed0cc2
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Mon Feb 4 13:42:51 2019 +0000

    configs: Add V4L2 codec driver to bcmrpi3_defconfig
    
    As this is now fixed to work with 64bit kernels, add it to the
    defconfig.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index 80691b1048ad..474d5c56a881 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -1071,6 +1071,7 @@ CONFIG_FB_FLEX=m
 CONFIG_FB_TFT_FBTFT_DEVICE=m
 CONFIG_SND_BCM2835=m
 CONFIG_VIDEO_BCM2835=m
+CONFIG_VIDEO_CODEC_BCM2835=m
 CONFIG_MAILBOX=y
 CONFIG_BCM2835_MBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1820_8882d4f278f00f73e898542740fbba6614ed0cc2.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1820_8882d4f278f00f73e898542740fbba6614ed0cc2.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
